zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/JobRequestValidator.java

```diff
@@ -60,7 +60,7 @@ public static Optional<ProblemDetail> validateJobUpdateRequest(
       final JobUpdateRequest updateRequest) {
     final List<String> violations = new ArrayList<>();
     final JobChangeset changeset = updateRequest.getChangeset();
-    if (changeset.getRetries() == null && changeset.getTimeout() == null) {
+    if (changeset == null || (changeset.getRetries() == null && changeset.getTimeout() == null)) {
```

Comment:
🔧 This validation seems wrong to me. It will add an error message if even one of the parameters (retries and timeout) are `null`. I think we only need to trigger it when both are missing.

Reply:
We add the violations if the changeset is null OR if both fields are null. There are also tests that verify this

Reply:
Yup, I saw them in the IT tests. I had my logic in reverse, good point.